### PR TITLE
CDAP-14477 Remove bucket from GCS Create/Delete action

### PR DESCRIPTION
**Pull request by rohitsinha54**
Issue: https://issues.cask.co/browse/CDAP-14477

- Remove Bucket from GCS Create and Delete Action
- Support create delete across multiple buckets by just taking paths

### CODE REVIEW

File: src/main/java/co/cask/gcp/gcs/actions/GCSBucketCreate.java [hunk 1]
```diff
@@ -126,20 +123,22 @@ public void run(ActionContext context) throws Exception {
...
     @Macro
-    public String failIfExists;
+    private String failIfExists;
+
+    public List<String> getPaths() {
+      return Arrays.stream(paths.split(",")).map(String::trim).collect(Collectors.toList());
+    }
+
+    public boolean getFailIfExists() {
+      return failIfExists.equalsIgnoreCase("yes");
```
Comment (albertshau) [inline src/main/java/co/cask/gcp/gcs/actions/GCSBucketCreate.java]: looks like there's a mismatch between the widget and this, where the widget has True and False but this... expects 'yes'. This is really weird, don't know why this plugin decided to be different. Let's change it from a String into a Boolean.

**Review by albertshau — COMMENTED** [review]
logic looks good and cleaner. One comment about a pre-existing bug

File: src/main/java/co/cask/gcp/gcs/actions/GCSBucketCreate.java [hunk 2]
```diff
@@ -126,20 +123,22 @@ public void run(ActionContext context) throws Exception {
...
+    @Description("Comma separated list of objects to be created.")
     @Macro
-    public String paths;
+    private String paths;
 
     @Name("failIfExists")
     @Description("Fail if path exists.")
     @Macro
-    public String failIfExists;
+    private String failIfExists;
```
Comment (albertshau) [inline src/main/java/co/cask/gcp/gcs/actions/GCSBucketCreate.java]: this should be a Boolean now.

Comment (rohitsinha54) [reply src/main/java/co/cask/gcp/gcs/actions/GCSBucketCreate.java]: Oops!!

**Review by albertshau — APPROVED** [review]
lgtm

**Pull request merged**